src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -204,40 +220,42 @@ export function EmbeddedPositionSearchPanel({
         const defSlotProps = {
             searchSummary: {
                 title: undefined,
-                searchType, coords, radius, fileName, rows,
                 showCoords: false,
                 showRadius: false,
                 showFileInfo: false,
                 sx:{}
             },
         };
-        
+
         return (
             isOpen ?
                 <div/>:
-                <Stack width={'100%'} >
+                <Stack p={0}>
                     <FormPanel
                         onSuccess={(request) => doSearch(request)}
                         direction={'row'}
                         width={'100%'}
                         slotProps={{
-                            searchBar: {px:0, py:0, justifyContent: 'left'},
+                            searchBar: {p:0, justifyContent: 'right'},
                         }}
                         cancelText=''>
-                        <Slot component={SearchSummary} {...defSlotProps.searchSummary} slotProps={slotProps?.searchSummary}/>
+                        <Stack {...{width:'100%', alignItems:'center'}}>
+                            <Slot component={SearchSummary} {...defSlotProps.searchSummary} searchType={searchType}
+                                  coords={coords} radius={radius} fileName={fileName}
+                                  rows={rows} slotProps={slotProps?.searchSummary}/>
```

Comment:
This is exactly the same as when you left them in defSlotProps.  Why move them out here?